include/onnxruntime/core/session/onnxruntime_cxx_inline.h

```diff
@@ -510,6 +510,21 @@ inline SessionOptions& SessionOptions::AppendExecutionProvider_TensorRT(const Or
   return *this;
 }
 
+inline SessionOptions& SessionOptions::SetCustomCreateThreadFn(CustomCreateThreadFn custom_create_thread_fn) {
```

Comment:
There is no wrappers for SetGlobalXXX methods. I am not sure where they should go.

Reply:
That's right - customer call into those API(s) directly by OrtApi, sample code is in test_main.cc of this PR.